src/Avalonia.Controls/Primitives/AccessText.cs

```diff
@@ -10,7 +11,7 @@ namespace Avalonia.Controls.Primitives
     /// <summary>
     /// A text block that displays a character prefixed with an underscore as an access key.
     /// </summary>
-    public class AccessText : TextBlock
+    public class AccessText : TextPresenter
```

Comment:
I don't think `AccessText` should inherit from `TextPresenter`. `TextPresenter` is meant to display the contents of a `TextBox`; `AccessText` isn't a text box (doesn't accept input), it's just a `TextBlock` that displays an underline when `ShowAccessKey` is set.

Reply:
Will revert the change. Makes sense👍